docs/source/uploading_files.rst

```diff
@@ -0,0 +1,119 @@
+==========================
+Uploading Referenced Files
```

Comment:
Same comment as on the other PR, this title may create confusion with the metadata type ReferenceFile